.github/renovate.json5

```diff
@@ -0,0 +1,46 @@
+{
+  // Configuration file for RenovateBot: https://docs.renovatebot.com/configuration-options
+  extends: ["config:base"],
+  labels: ["dependencies"],  // For convenient searching in GitHub
+  packageRules: [
+    {
+      // Automerge patches, pin changes and digest changes.
+      // Also groups these changes together.
+      groupName: "Default rules",
+      matchUpdateTypes: ["patch", "pin", "digest"],
+      automerge: true
+    },
+    {
+      // Minor changes can be automerged for dev dependencies, but are also deprioritised.
+      groupName: "Development dependencies",
+      matchDepTypes: ["devDependencies"],
+      matchUpdateTypes: ["minor", "patch", "pin", "digest"],
+      prPriority: -1,
+      automerge: true
+    },
+    {
+      // Update all craft-* packages in one higher-priority PR
+      groupName: "Craft packages",
+      matchPackagePrefixes: ["craft-"],
+      matchLanguages: ["python"],
+      prPriority: 10,
+    }
+  ],
+  regexManagers: [
+    {
+      // tox.ini can get updates too if we specify for each package.
+      fileMatch: ["tox.ini"],
+      matchStrings: [
+        "# renovate: datasource=(?<datasource>\\S+)\n\\s+(?<depName>.*?)==(?<currentValue>.*?)\\n"
+      ]
+    }
+  ],
+  timezone: "Etc/UTC",
+  automergeSchedule: "after 1 am and before 7 am",
+  schedule: "every weekend",
+  prConcurrentLimit: 5, // No more than 5 open PRs at a time.
+  prCreation: "not-pending", // Wait until status checks have completed before raising the PR
+  prNotPendingHours: 4, // ...unless the status checks have been running for 4+ hours.
+  prHourlyLimit: 4, // No more than 4 PRs per hour.
+  stabilityDays: 2 // Wait 2 days from release before updating.
+}
```

Comment:
can you please add a newline here? Github likes to complain about the lack of it

Reply:
Done!